ams/ams-server/src/main/java/com/netease/arctic/ams/server/optimize/OptimizeTaskItem.java

```diff
@@ -87,6 +94,18 @@ public void onPending() {
     try {
       Preconditions.checkArgument(optimizeRuntime.getStatus() != OptimizeStatus.Prepared,
           "task prepared, can't on pending");
+
+      // can update max execute time on optimizing
```

Comment:
We should set the `max-execute-time` after tasks are polled from the queue

Reply:
ok